UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -499,7 +545,17 @@ public class Background : UndertaleObject, INotifyPropertyChanged
         /// The y coordinate of the background in the room.
         /// </summary>
         public int Y { get => _y; set { _y = value; OnPropertyChanged(); UpdateStretch(); } }
+
+        /// <summary>
+        /// Whether this <see cref="Background"/> is tiled horizontally.<br/>
+        /// <c>0</c> - <see langword="false"/>, <c>1</c> - <see langword="true"/>.
```

Comment:
A) Because in GameMaker it is a default state.
B) Yes, it's used by GameMaker; didn't test, but I believe it will treat that as `1`.
C) Agree.

Reply:
> but I believe it will treat that as 1.

Would like to have proper testing for this. Again, probably not for this PR, but if its possible via GM to somehow set this attribute to anything other than 0 and 1 (with GM still compiling it), we may wanna handle it properly as well then. Also to properly know if the games will treat it as `1`, or just as anything else.

Reply:
So, should I make a separate PR only for making them `private`?

Reply:
That would depend on the testing from above.

Reply:
Just tested that - any value that ≥1 is treated as `1`.

Reply:
> That would depend on the testing from above.

I meant that I can do it right here

Reply:
Nah, make a separate PR for it.